tasks/mpi/shuravina_o_contrast/perf_tests/main.cpp

```diff
@@ -0,0 +1,67 @@
+#include <gtest/gtest.h>
+
+#include <chrono>
+#include <cstddef>
+#include <cstdint>
+#include <memory>
+#include <vector>
+
+#include "core/perf/include/perf.hpp"
+#include "core/task/include/task.hpp"
+#include "mpi/shuravina_o_contrast/include/ops_mpi.hpp"
+
+TEST(shuravina_o_contrast_mpi, test_pipeline_run) {
+  constexpr size_t kSize = 2048;
```

Comment:
Please, align the numbers in mpi and seq versions
```suggestion
  constexpr size_t kSize = 4096;
```